src/VisualQueryBuilder/components/OperationEditor.tsx

```diff
@@ -0,0 +1,336 @@
+import { css, cx } from '@emotion/css';
+import React, { useEffect, useState } from 'react';
+import { Draggable } from 'react-beautiful-dnd';
+import { v4 } from 'uuid';
+
+import { DataSourceApi, GrafanaTheme2, TimeRange } from '@grafana/data';
+import { Button, Icon, InlineField, Tooltip, useTheme2 } from '@grafana/ui';
+
+import { OperationHeader } from './OperationHeader';
+import {
+  QueryBuilderOperation,
+  QueryBuilderOperationDefinition,
+  QueryBuilderOperationParamDef,
+  QueryBuilderOperationParamValue,
+  VisualQueryModeller,
+} from '../types';
+import { Stack } from '../../QueryEditor/Stack';
+import { getOperationParamEditor, getOperationParamId } from './OperationParamEditor';
+
+interface Props {
+  operation: QueryBuilderOperation;
+  index: number;
+  query: any;
```

Comment:
We could also make this a generic like src/VisualQueryBuilder/components/OperationList.tsx

Reply:
Fixed in https://github.com/grafana/grafana-experimental/pull/98/commits/a594bdbf7277670de3fe92f74d8da86212959bc9